doc/doxygen/header.html

```diff
@@ -32,12 +32,25 @@
     <![endif]-->
         <script>
             function displaySR(){
+            searchBox.Search();
             var rhtml=(document.getElementById('MSearchResults').contentWindow.document.body.innerHTML).replace(/href=\"\.\.\//g,'href="');
             document.getElementById('MSearchResultsWindow').style.display='none';
```

Comment:
Now it fails here :-/ Seems like the JavaScript has bit rotten a bit. Anyway, that is not the concern of this PR. I'd say this can go in. It should still work on the live system, which is more important.